BotRoyaleUnity/Assets/Scripts/PartHealth.cs

```diff
@@ -48,6 +48,7 @@ public void SubtractHealth(float damage)
                     subtractedStrength = true;
                     Destroy(gameObject);
                     GameStateManager.Instance.killRobot(gameObject);
+                    socketIO.EmitCurrentParts(transform.parent.gameObject);
```

Comment:
I'm assuming EmitCurrentParts needs to be called when any part of the robot takes damage, but I think this would make it so it's only called when a part gets destroyed

Reply:
I think we should leave it like this for performance reasons unless we implement the taking damage in the react-battle screen.

Reply:
makes sense